source/jormungandr/jormungandr/parking_space_availability/car/car_park_provider_manager.py

```diff
@@ -42,7 +43,7 @@ def __init__(self, car_park_providers_configurations):
             self.car_park_providers.append(self._init_class(configuration['class'], arguments))
 
     def _handle_poi(self, item):
-        if 'poi_type' in item and item['poi_type']['id'] == POI_TYPE_ID:
+        if 'poi_type' in item and item['poi_type']['id'] in POI_TYPE_ID:
```

Comment:
Here, without an array of ids I think we can have weird bugs like `poi_type:amenity:park` would be valid, as we check that a string is in another one, no?

Reply:
True